include/simdjson.h

```diff
@@ -5,6 +5,33 @@
  * @mainpage
  *
  * Check the [README.md](https://github.com/lemire/simdjson/blob/master/README.md#simdjson--parsing-gigabytes-of-json-per-second).
+
```

Comment:
Having an example on the front page sounds great!

I think we want to keep the * in front of these lines, so that people can see what's code and what's comments. markdown  lets you put code in if you indent it (so add four spaces, like `  *     #include "simdjson.h"`

Also, period after "examples."

Reply:
I updated the comment, not sure if that's the way it should be done. Can you please show what you mean if last update isn't correct.

Reply:
Yep! Thanks!